### PR TITLE
Add dockerfile

### PR DESCRIPTION
**Pull request by joram**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    python3 -m ensurepip && \
+    rm -r /usr/lib/python*/ensurepip && \
+    pip3 install --upgrade pip setuptools && \
+    if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi && \
+    if [[ ! -e /usr/bin/python ]]; then ln -sf /usr/bin/python3 /usr/bin/python; fi && \
+    rm -r /root/.cache
+
+
+RUN mkdir /battlesnakeio_play
+RUN cd /battlesnakeio_play
```
Comment (dlsteuer) [inline Dockerfile]: This line is redundant, and just adds an extra layer to the dockerfile;

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+RUN apk add --no-cache python3 && \
+    python3 -m ensurepip && \
+    rm -r /usr/lib/python*/ensurepip && \
+    pip3 install --upgrade pip setuptools && \
+    if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi && \
+    if [[ ! -e /usr/bin/python ]]; then ln -sf /usr/bin/python3 /usr/bin/python; fi && \
+    rm -r /root/.cache
+
+
+RUN mkdir /battlesnakeio_play
```
Comment (dlsteuer) [inline Dockerfile]: don't need to create the folder before using, copy will create for you. Also what do you think about just using a generic /app folder in the container.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ENV PYTHONPATH="${PYTHONPATH}:/battlesnakeio_play"
+ENV DJANGO_SETTINGS_MODULE="settings.settings"
+WORKDIR /battlesnakeio_play
+
+ADD requirements.txt /battlesnakeio_play/requirements.txt
+RUN pip install -r /battlesnakeio_play/requirements.txt
+
+COPY ./play /battlesnakeio_play/
+
+RUN ls -r /battlesnakeio_play
```
Comment (dlsteuer) [inline Dockerfile]: remove this, adds another layer that is not required.

File: play/settings/settings.py [hunk 4]
```diff
@@ -79,14 +88,21 @@
...
 
+DB_NAME = get_env("POSTGRES_DB", "battlesnakeio_play", True)
+DB_USER = get_env("POSTGRES_USER", None, False)
+DB_PASS = get_env("POSTGRES_PASSWORD", None, False)
+DB_HOST = get_env("BATTLESNAKEIO_POSTGRES_HOST", None, False)
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
```
Comment (dlsteuer) [inline play/settings/settings.py]: what do you think about running just sqlite locally, and using postgres in the cluster?

File: Dockerfile (hunk 2 above)
Comment (joram) [reply Dockerfile]: `/app` seems fine to me, probably more of a standard than project name, I will switch

File: play/settings/settings.py (hunk 4 above)
Comment (joram) [reply play/settings/settings.py]: seems sensible. I will make it mounted in a volume so we can persist data locally.

**Comment by codeallthethingz** [comment]
John says "Daniel can you do it?"

**Pull request merged**